kitchen.yml

```diff
@@ -130,7 +130,6 @@ suites:
     controls:
     - keychain-creation
     - login-keychain-creation
-    - default-keychain-creation
```

Comment:
Was there an unrelated test failure here?

Reply:
There isn't a `default-keychain-creation` control. Aaron's work to detect this issue caused this to appear as a test failure instead of being silently ignored.